packages/venia-styleguide/src/components/Heading/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Heading from './Heading';
+
+export default Heading;
+export const H1 = props => <Heading {...props} level={1} />;
+export const H2 = props => <Heading {...props} level={2} />;
+export const H3 = props => <Heading {...props} level={3} />;
+export const H4 = props => <Heading {...props} level={4} />;
+export const H5 = props => <Heading {...props} level={5} />;
+export const H6 = props => <Heading {...props} level={6} />;
```

Comment:
These don't seem to be used anywhere. Are these supposed to be components for the styleguide UI or for use in PWA Studio (why not both?)

Reply:
Oh nvm I see the `MDXAdapter` :)